README.md

```diff
@@ -1,6 +1,6 @@
 # Unsplash Wallpaper
 
-This is a script to set unsplash wallpapper using unsplash api to get random images or random user images likes.
+This is a script to set you linux box wallpapper using Unsplash api to get a random image or random image that the logged in user likes.
```

Comment:
image that the logged in user likes, it's not required user logged.

Reply:
Done :)